astropy/modeling/core.py

```diff
@@ -1074,22 +1074,28 @@ def evaluate(self, *args, **kwargs):
 
     def render(self, out=None, coords=None):
         """
-        Evaluates a model on an input array. Evaluation is limited to
-        a bounding box if the `Model.bounding_box` attribute is set.
+        Evaluates a model on an input array.  I.e., "discretizes" the model.
+        The key difference relative to evaluating the model directly is that
+        this method is limited to a bounding box if the `Model.bounding_box`
+        attribute is set.
 
         Parameters
         ----------
         out : `numpy.ndarray`, optional
-            The array on which the model is to be evaluated.
+            An array that the evaluated model will be added to.  If this is not
+            given (or given as ``None``), a new array will be created.
         coords : array-like, optional
-            Coordinate arrays mapping to ``arr``, such that
-            ``arr[coords] == arr``.
+            An array to be used to translate from the model's input coordinates
+            to the ``out`` array. It should have the property that
+            ``self(coords)`` yields the same shape as ``out``.  If ``out`` is
+            not specified, ``coords`` will be used to determine the shape of the
+            returned array.
```

Comment:
I think this could still be clarified further--that by default the coordinate grid to evaluate the model on is determined by the bounding box.